cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -934,14 +934,47 @@ search = `cs_wineventlog_security` EventCode IN (627, 628, 4723, 4724, 4794) \
 | `cs_generate_time_fields_for_filter(time)` \
 | where time_wday="Saturday" OR time_wday="Sunday" OR time_hour<6 OR time_hour>19 \
 | `cs_remove_time_fields_for_filter(time)` \
-| table _time, user, Account_Domain, Account_Name, msad_action, action, EventCode \
+| table _time, ComputerName, Target_Account_Name, Target_Account_Domain, Subject_Account_Name Subject_Account_Domain Caller_Workstation, msad_action, action, EventCode \
 | `cs_ad_password_change_outside_working_hour_filter`
 action.summary_index = 1
 action.summary_index._name = cyences
 action.summary_index.category = AD
 action.summary_index.alert_name = AD - Password Change Outside Working Hour
 
 
+[AD - Multiple Password Changes in Short Time Period]
+disabled = 1
+enableSched = 1
+alert.track = 1
+alert.severity = 4
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule = 59 * * * *
+description = This alert will show if user made multiple password changes in the last hours. \
+\
+Data Collection - Splunk_TA_windows - WinEventLog://Security stanza
+dispatch.earliest_time = -62m@m
+dispatch.latest_time = -2m@m
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = `cs_wineventlog_security` EventCode IN (627, 628, 4723, 4724, 4794) \
+| table _time, ComputerName, Target_Account_Name, Target_Account_Domain, Subject_Account_Name Subject_Account_Domain Caller_Workstation, msad_action, action, EventCode \
```

Comment:
Only one table (last one) command is required for data formatting.

Reply:
If we remove first table then stats will do values(*) for all the fields so to limit the fields first table the required fields. and second table is to show fields in the specific format.

Reply:
Use fields command.